dotnet/src/Connectors/Connectors.AzureAISearch.UnitTests/AzureAISearchVectorStoreRecordCollectionTests.cs

```diff
@@ -608,16 +607,15 @@ public async Task CanSearchWithTextAndFilterAsync()
         var filter = new VectorSearchFilter().EqualTo(nameof(MultiPropsModel.Data1), "Data1FilterValue");
 
         // Act.
-        var searchResults = await sut.SearchAsync(
-            VectorSearchQuery.CreateQuery(
-                "search string",
-                new()
-                {
-                    Limit = 5,
-                    Offset = 3,
-                    Filter = filter,
-                    VectorFieldName = nameof(MultiPropsModel.Vector1)
-                }),
+        var searchResults = await sut.VectorizableTextSearchAsync(
```

Comment:
A bit confused here why we have two names different, I prefer `Vectorized`.

`Vectorized` / `Vectorizable` ?

```suggestion
        var searchResults = await sut.VectorizableTextSearchAsync(
```

Reply:
Vectorized means that you are doing a search with a vector, as in the data is already vectorized.  Vectorizable means that you have something else that is vectorizable and will be vectorized lower down the stack, e.g. text.
Azure AI Search uses the same nomanclature, but we will probably revisit this before we go live again.

See e.g. the following for azure ai search.
https://learn.microsoft.com/en-us/javascript/api/@azure/search-documents/vectorizabletextquery?view=azure-node-latest